test/WebJobsStartupTests/WebJobsStartupTests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Note that I added a new test project so we can test this Startup stuff end-to-end in a real application.